rclpy/rclpy/utilities.py

```diff
@@ -39,6 +39,12 @@ def get_default_context(*, shutting_down=False):
         return g_default_context
 
 
+def get_default_domain_id():
```

Comment:
maybe we could add test in https://github.com/ros2/rclpy/blob/master/rclpy/test/test_utilities.py.

Reply:
OK. I will supplement test codes.